src/Symfony/Bridge/Twig/DataCollector/TwigDataCollector.php

```diff
@@ -44,6 +44,17 @@ public function collect(Request $request, Response $response, \Exception $except
     /**
      * {@inheritdoc}
      */
+    public function reset()
+    {
+        // TODO: Twig PR pending. Update dependencies accordingly.
```

Comment:
can be done now to me

Reply:
PR on Twig has been merged now and new releases are out for both 1.x and 2.x versions.

Reply:
Dependencies updated, comment removed.